js-intermediate.test.js

```diff
@@ -3,24 +3,44 @@
 
 // 1. Write the code that would make this test pass.
 
-describe("multTwo", () => {
-    test("returns an array with all the numbers multiplied by two", () => {
-        expect(multTwo([3, 4, 5])).toEqual([6, 8, 10])
-        expect(multTwo([23, -9, 0])).toEqual([46, -18, 0])
-        expect(multTwo([4.5, -4.5, 12])).toEqual([9, -9, 24])
-    })
-})
+// test("returns an array with all the numbers multiplied by two", () => {
+      
+        // expect(multTwo([3, 4, 5])).toEqual([6, 8, 10])
+        // expect(multTwo([23, -9, 0])).toEqual([46, -18, 0])
+        // expect(multTwo([4.5, -4.5, 12])).toEqual([9, -9, 24])
+//     })
+// })
+
+let friendlyArray = [1,2,3,4,5]
+let newArr = []
+const multTwo = (arr) => {
+  for (let i = 0; i < arr.length; i++)
+  newArr.push(arr[i] *2 )
+  return newArr;
+};
+console.log(multTwo(friendlyArray))
 
```

Comment:
Remember, the point of testing is we don't have to create a variable or log the code. The test is going to do that for us.